src/index.html

```diff
@@ -1,17 +1,196 @@
 <!doctype html>
 <html lang="en">
+
   <head>
     <meta charset="UTF-8">
-    <meta
-      name="viewport"
+    <meta name="viewport"
       content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
     >
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>HTML Form</title>
     <link rel="stylesheet" href="./style.css">
   </head>
+
   <body>
-    <h1>HTML Form</h1>
+
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+
+      <fieldset class="fieldset">
+
+        <legend>Personal information</legend>
+
+        <label class="field">
+          Surname:
+          <input
+          class="field-form"
+          name="surName"
+          type="text"
+          autocomplete="off"
+          required
+          >
```

Comment:
attributes that are moved to a new line should have 2-spaces indent in order it is easier to read your code
```suggestion
          <input
            class="field-form"
            name="surName"
            type="text"
            autocomplete="off"
            required
          >
```